packages/siwe/src/provider.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { AuthResponse, NonceResponse, SiweFetchers } from './requests'
+import { Config, useEthers } from '@usedapp/core'
+import {
+  setupTestingConfig,
+  TestingNetwork,
+  IdentityWrapper,
+  renderDAppHook,
+  getWaitUtils,
+} from '@usedapp/core/testing'
+import { SiweProvider, useSiwe } from './provider'
+import React, { useEffect } from 'react'
+import { expect } from 'chai'
+import { SiweMessage } from 'siwe'
+import 'mock-local-storage'
+
+let store: any = {}
```

Comment:
Looks like the types are known

```suggestion
let store: Record<string, string> = {}
```